docs/reference/data-streams/set-up-a-data-stream.asciidoc

```diff
@@ -131,8 +131,9 @@ this pattern.
 ----
 ====
 
-* A `data_stream` definition containing `@timestamp` in the `timestamp_field`
-property. The `@timestamp` field must be included in every document indexed to
+* A `data_stream` definition, optionally containing `@timestamp` in the `timestamp_field`
+property. If the `@timestamp` field definition is omitted {es} will default to it.
+The `@timestamp` field must be included in every document indexed to
```

Comment:
I'm not sure if there is any advantage to including a body in the `data_stream` object anymore. I think it's easier to just tell users to supply an empty body.
```suggestion
* A `data_stream` object with an empty body (`{ }`). 
```

The timestamp requirements are already included in the prerequisites:
https://www.elastic.co/guide/en/elasticsearch/reference/master/set-up-a-data-stream.html#data-stream-prereqs

We may just want to re-emphasize in the index example shown later on this page.

We'll also need to update the index template examples. I can do both of those in a follow-up PR.